bhdr.js

```diff
@@ -2,10 +2,9 @@
 
 const Histogram = require('native-hdr-histogram')
 const histutils = require('hdr-histogram-percentiles-obj')
-const chalk = require('chalk')
-const colors = ['red', 'green', 'yellow', 'blue', 'magenta', 'cyan', 'white', 'gray']
 const console = require('console') // for proxyquire in tests
 const process = require('process') // for proxyquire in tests
```

Comment:
when es6 modules land in node 10 - proxyquire is screwed - espcially if console and process are implemented in core using es6 modules (es6 mods have immutable cache)